src/modules/dashboard/components/NotEnoughReputation/NotEnoughReputation.tsx

```diff
@@ -1,37 +1,54 @@
+import { ROOT_DOMAIN_ID } from '@colony/colony-js';
 import React from 'react';
 import { defineMessage, FormattedMessage } from 'react-intl';
 
+import { getMainClasses } from '~utils/css';
 import DialogSection from '~core/Dialog/DialogSection';
 
 import styles from './NotEnoughReputation.css';
 
+interface Appearance {
+  marginTop?: 'negative';
+}
+
+interface Props {
+  domainId?: number;
+  appearance?: Appearance;
+}
+
 const MSG = defineMessage({
   title: {
     id: 'NotEnoughReputation.title',
-    defaultMessage:
-      'There is not enough reputation in colony/domain to create this motion.',
+    defaultMessage: `There is no reputation in this {onRootDomain, select,
+      true {colony}
+      false {domain}
```

Comment:
Should be `team` here as we don't use domains in the copies.

Reply:
Good catch! :1st_place_medal: